### ISSUE TITLE
Add brace-style option for stroustrup

### ISSUE THREAD

**Issue by nzakas** [eslint/eslint#645]
The `brace-style` rule should allow you to specify which brace style to validate against. The default should be "1tbs" (current), but we should at least also have "stroustrup" (http://en.wikipedia.org/wiki/Indent_style#Variant:_Stroustrup), which is essentially the same as 1tbs except that the `else` is on a separate line from the precending `}`, such as:

``` js
if (foo) {

}
else {

}
```
